mods/ra/weapons/ballistics.yaml

```diff
@@ -114,10 +114,17 @@ TurretGun:
 155mm:
 	Inherits: ^Artillery
 	MinRange: 4c0
+	Warhead@1Dam: SpreadDamage
+		Falloff: 100, 55, 20, 5
+		Versus:
+			None: 40
+			Light: 35
 	Report: tank5.aud
 	TargetActorCenter: true
 	Projectile: Bullet
 		ContrailLength: 30
+		Speed: 170
+		Inaccuracy: 0c614
```

Comment:
Not a fan of the artillery changes. This makes them too similar to V2s in my opinion, and the exact opposite of what they are supposed to be.

Reply:
I agree here in principle - the Artillery vs V2 contrast has been as much a part of OpenRA as the flak truck or allied hind - but we should be open to evolving this if needs be.

Considering the negative feedback so far and the big unknown impact that the fog targeting changes will bring, I think it would be safer to defer this to a future release.